Plot/src/main/java/io/deephaven/plot/util/tables/ColumnHandlerFactory.java

```diff
@@ -117,7 +118,7 @@ private ColumnHandlerTable(final Table table, final String columnName, Class typ
 
         protected DataColumn getDataColumn() {
             if (dataColumn == null) {
-                dataColumn = table.getColumn(columnName);
+                dataColumn = ((BaseTable) table).getColumn(columnName);
```

Comment:
These will all be raw types warnings. Want to do a search replace to `(BaseTable<?>)`?